src/main/common/utils.h

```diff
@@ -55,7 +55,11 @@
 #define PP_CALL(macro, ...) macro(__VA_ARGS__)
 
 #if !defined(UNUSED)
-#define UNUSED(x) (void)(x)
+#define UNUSED(x) (void)(x) // Variables and parameters that are not used
+#endif
+
+#if !defined(DISCARD)
```

Comment:
Is `DISCARD()` already defined somewhere else? If not, we should not guard it here - we'd rather get an error if and when another definition was introduced (say as part of a library header), and then could examine the new definition to make sure it had the functionality we want.

`UNUSED()` is guarded because it is defined in some libraries as well, and the guard was added after vetting that none of the other defines introduced functionality that we don't want.

Reply:
Ah. Didn't think of the case.